exam/server_client.ipynb

```diff
@@ -32,7 +32,7 @@
     }
    ],
    "source": [
-    "from uniflow.client import Client\n",
+    "from uniflow.model.client import Client\n",
```

Comment:
model should not have a client. client should belong to either extract or transform

Reply:
Updated with refactor commit